lib/elixir/lib/module.ex

```diff
@@ -725,7 +738,10 @@ defmodule Module do
       end
 
   """
-  def defines?(module, tuple) when is_tuple(tuple) do
+  @spec defines?(module, function_arity) :: boolean
+  def defines?(module, {function_macro_name, arity} = tuple)
```

Comment:
Indeed, we use `function_name` in same context in `make_overridable`, and `overridable?`.